specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabric/preview/2022-08-01-preview/nodetype.json

```diff
@@ -0,0 +1,1564 @@
+        }
+      },
+      "description": "Describes the frontend configurations for the node type."
+    },
+    "IPAddressType": {
+      "type": "string",
+      "description": "The IP address type.\n",
+      "enum": [
+        "IPv4",
+        "IPv6"
+      ],
+      "default": "IPv4",
+      "x-ms-enum": {
+        "name": "IPAddressType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "IPv4",
+            "description": "IPv4 address type."
+          },
+          {
+            "value": "IPv6",
+            "description": "IPv6 address type."
+          }
+        ]
+      }
+    },
+    "ManagedProxyResource": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Azure resource identifier.",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "Azure resource name.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "Azure resource type.",
+          "readOnly": true
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Azure resource tags.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData"
+        }
+      },
+      "description": "The resource model definition for proxy-only resource.",
+      "x-ms-azure-resource": true
+    },
+    "NodeType": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ManagedProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NodeTypeProperties",
+          "description": "The node type properties"
+        },
+        "sku": {
+          "$ref": "#/definitions/NodeTypeSku",
+          "description": "The node type sku."
+        }
+      },
+      "description": "Describes a node type in the cluster, each node type represents sub set of nodes in the cluster."
+    },
+    "NodeTypeActionParameters": {
+      "type": "object",
+      "required": [
+        "nodes"
+      ],
+      "properties": {
+        "nodes": {
+          "description": "List of node names from the node type.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "force": {
+          "description": "Force the action to go through.",
+          "type": "boolean"
+        },
+        "udWiseUpdate": {
```

Comment:
>udWiseUpdate [](http://example.com/codeflow?start=9&length=12)

Consider if you can use enum over boolean, see [https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html)

"Replace boolean/switch properties with better enum"

Reply:
Ok thanks for the suggestion, We will remove this parameter from this api and introduce it next month on a new api version as an enum